README.md

```diff
@@ -1,4 +1,4 @@
-# datalab
+# datalab [![Build Status](https://travis-ci.org/googledatalab/pydatalab.svg?branch=master)](https://travis-ci.org/googledatalab/pydatalab) [![PyPI Package](https://img.shields.io/pypi/v/datalab.svg)](https://pypi.python.org/pypi/datalab/1.0.1)
```

Comment:
I noticed that version 1.0.1 is hardcoded in the pypi link. Could we link directly to https://pypi.python.org/pypi/datalab/ instead?